app/src/main/java/com/devindi/wallpaper/model/map/MapModule.kt

```diff
@@ -9,14 +9,14 @@ import org.osmdroid.tileprovider.cachemanager.CacheManager
 import org.osmdroid.tileprovider.modules.IFilesystemCache
 import org.osmdroid.tileprovider.tilesource.OnlineTileSourceBase
 
-val mapModule: Module = module {
-    single { SqlTileCache(get()) as IFilesystemCache }
-    single { createWallpaperHandler(WallpaperManager.getInstance(androidApplication())) }
+val mapModule = module {
+    scope("map_scope") { SqlTileCache(get()) as IFilesystemCache }
+    scope("map_scope") { createWallpaperHandler(WallpaperManager.getInstance(androidApplication())) }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/a4a41430d88882963b009b10c19d56ec5d7b1f46/app/src/main/java/com/devindi/wallpaper/model/map/MapModule.kt#L14"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>